src/Process/CommandExecutor.php

```diff
@@ -43,4 +43,11 @@ public function passthru(string $command, OutputInterface $output): int
         });
         return $process->getExitCode();
     }
+
+    public function executeDirectly(string $command, bool $throwOnError = false): Process
```

Comment:
```suggestion
    public function executeDirectly(string $command): Process
```

remove throwOnError if it is not used anywhere